### PR TITLE
feat: header bar

### PR DESCRIPTION
**Pull request by banahaker**
**[ NEW FEATURE ] Home Page completed, include below:**
 - header bar
 - home page

*Please CODE REVIEW and merge to develop branch*

### CODE REVIEW

**Comment by banahaker** [comment]
The header bar has been completed, but the content of home page hasn't yet!

**TBD**

**Comment by banahaker** [comment]
I finish the home page basic design, then I notice that I hadn't given header bar a responsive design(RWD), so I'm going on!!

**TBD**

**Comment by banahaker** [comment]
I FINISHED !!!!!!!!!
@SiongSng PTAL!!

**Comment by banahaker** [comment]
I got a eslint error, and I already fixed it.

File: src/views/page_home.vue [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<script lang="ts" setup>
+import block_headerVue from '@/components/block_header.vue';
+import block_home_introVue from '@/components/block_home_intro.vue';
```
Comment (SiongSng) [inline src/views/page_home.vue:3]:
```suggestion
import blockHomeIntroVue from '@/components/block_home_intro.vue';
```

File: src/views/page_home.vue [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+</script>
+
 <template>
-  <div>home</div>
-</template>
\ No newline at end of file
+  <div id="home">
+    <block_header-vue></block_header-vue>
```
Comment (SiongSng) [inline src/views/page_home.vue:8]:
```suggestion
    <blockHeaderVue></blockHeaderVue>
```

**Review by SiongSng — COMMENTED** [review]
Please use camel case.

File: src/views/page_home.vue [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+    <block_home_intro-vue></block_home_intro-vue>
```
Comment (banahaker) [inline src/views/page_home.vue]: um...在html裡面用component要全部用 `-` 來連接
Such as `<block-home-intro-vue/>`

**Review by SiongSng — APPROVED** [review]
LGTM

**Pull request merged**